### PR TITLE
docs(examples): standardize unified code format and fix some type

### PR DESCRIPTION
**Pull request by warmthsea**
#### Tnvolving component examples
image

#### 1、Standardize unified code format 
- add newline

image

- tag add newline, best readability

image

#### 2、add some examples type

image
image
image

#### 3、fix some error code

image

### CODE REVIEW

**Review by warmthsea — COMMENTED** [review]
LGTM

**Review by btea — APPROVED** [review]
Thanks!

**Review by tolking — COMMENTED** [review]
I suggest that you prioritize using the basic types exported from the component library

```diff
- const size = ref<Required<DescriptionInstance>['size']>('default')
+ const size = ref<ComponentSize>('default')
```

**Comment by warmthsea** [comment]
> I suggest that you prioritize using the basic types exported from the component library
> 
> ```diff
> - const size = ref<Required<DescriptionInstance>['size']>('default')
> + const size = ref<ComponentSize>('default')
> ```

https://github.com/element-plus/element-plus/pull/16370/commits/4708d688941411bcef133fa7767611ef1b3d5148

**Comment by warmthsea** [comment]
Consider incorporating this PR?  
Or is there anything else that needs to be modified?

File: docs/examples/descriptions/sizes.vue [hunk 1]
```diff
@@ -88,23 +88,26 @@
...
-      >No.1188, Wuzhong Avenue, Wuzhong District, Suzhou, Jiangsu Province
+    <el-descriptions-item label="Address">
+      No.1188, Wuzhong Avenue, Wuzhong District, Suzhou, Jiangsu Province
     </el-descriptions-item>
   </el-descriptions>
 </template>
 
 <script setup lang="ts">
 import { computed, ref } from 'vue'
+import { ElDescriptions } from 'element-plus'
```
Comment (kooriookami) [inline docs/examples/descriptions/sizes.vue:99]:
```suggestion
```

File: docs/examples/page-header/additional-sections.vue [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <template>
-  <el-page-header :icon="null">
+  <el-page-header :icon="''">
```
Comment (kooriookami) [inline docs/examples/page-header/additional-sections.vue:2]: Why not `null`?

File: docs/examples/page-header/no-icon.vue [hunk 3]
```diff
@@ -1,5 +1,5 @@
 <template>
-  <el-page-header :icon="null">
+  <el-page-header :icon="''">
```
Comment (kooriookami) [inline docs/examples/page-header/no-icon.vue:2]: Same as above.

**Review by kooriookami — CHANGES_REQUESTED** [review]
I found some types you used `InstanceType` not `Props`. Why not use `Props` consistently

File: docs/examples/page-header/additional-sections.vue (hunk 2 above)
Comment (warmthsea) [reply docs/examples/page-header/additional-sections.vue]: Change to undefined.

screenshots

**Review by warmthsea — COMMENTED** [review]
Changes completed.

File: docs/examples/page-header/additional-sections.vue [hunk 4]
```diff
@@ -1,5 +1,5 @@
 <template>
-  <el-page-header :icon="null">
+  <el-page-header :icon="undefined">
```
Comment (kooriookami) [inline docs/examples/page-header/additional-sections.vue:2]: `undefined` will not take effect. So don't change it.

**Review by kooriookami — CHANGES_REQUESTED** [review]

File: docs/examples/page-header/additional-sections.vue (hunk 4 above)
Comment (warmthsea) [reply docs/examples/page-header/additional-sections.vue:2]: Don't want to change it? ´null´will cause a type error

Comment (warmthsea) [reply docs/examples/page-header/additional-sections.vue]: Has been changed back to null.
https://github.com/element-plus/element-plus/pull/16370/commits/279305a514de5f212929a1ff5a3a25aabe2e7d0c

Comment (kooriookami) [reply docs/examples/page-header/additional-sections.vue]: It's an error of prop types. May be fixed later.

**Review by kooriookami — APPROVED** [review]

**Pull request merged**